app/src/main/java/com/github/libretube/ui/adapters/PlaylistAdapter.kt

```diff
@@ -84,9 +84,7 @@ class PlaylistAdapter(
 
             if (!streamItem.uploaderUrl.isNullOrBlank()) {
                 channelContainer.setOnClickListener {
-                    streamItem.uploaderUrl?.toID()?.let {
```

Comment:
Redundant null safety.

Reply:
Caused by https://github.com/libre-tube/LibreTube/pull/2755, that might also be the case in some other places in the app now